.github/workflows/ci-matrix.yml

```diff
@@ -51,6 +57,11 @@ jobs:
         with:
           name: Node-${{ matrix.target.name }}
           path: results
+      - name: diagnostics
+        if: failure()
+        run: |
+          echo "final disc diagnostics ------>"
+          df -h ~
```

Comment:
What are you planning to do with these new lines? Do you think they should stay in or they rather played their role only in debugging that is already over now?